publications.json

```diff
@@ -3,6 +3,7 @@
       {
          "bio":"The Advocate promotes the dissemination of literature that examines youth-related issues with the goal of encouraging youth advocacy in local and federal politics.",
          "bioShort":"Encouraging political advocacy by examining issues relevant to college students",
+         "contentTypes":["articles"],
```

Comment:
can we get an explanation of what's going on in the publications.json in the PR description? I see a bunch of content types articles added as well as a magazine that isn't mentioned in the PR